sqlitex/pool.go

```diff
@@ -59,14 +59,37 @@ type Pool struct {
 }
 
 // Open opens a fixed-size pool of SQLite connections.
+//
 // A flags value of 0 defaults to:
 //
 //	SQLITE_OPEN_READWRITE
 //	SQLITE_OPEN_CREATE
 //	SQLITE_OPEN_WAL
 //	SQLITE_OPEN_URI
 //	SQLITE_OPEN_NOMUTEX
-func Open(uri string, flags sqlite.OpenFlags, poolSize int) (pool *Pool, err error) {
+func Open(uri string, flags sqlite.OpenFlags, poolSize int, initScripts ...string) (pool *Pool, err error) {
```

Comment:
initScripts is unused?

Reply:
Thank you. This was a botched refactor and I'm removing this parameter. If you want an initScript you should be forced to at least think about providing a context.

Reply:
```suggestion
func Open(uri string, flags sqlite.OpenFlags, poolSize int) (pool *Pool, err error) {
```
Remove unused parameter in sqlitex.Pool